packages/lib/src/components/Ach/components/AchInput/AchInput.tsx

```diff
@@ -204,6 +204,7 @@ export default AchInput;
 
 const extractPropsForSFP = (props: ACHInputProps) => {
     return {
+        allowedDOMAccess: props.allowedDOMAccess,
```

Comment:
Looks like the `allowedDOMAccess` got added again to the codebase. Is it intentional?

Reply:
I think there was a merge error. It should be fixed now